Student/StudentUITests/StudentUITestCase.swift

```diff
@@ -37,7 +37,9 @@ class StudentUITestCase: CoreUITestCase {
             let data = CFDataGetBytePtr(pixelData) else {
             return nil
         }
-        let red = UInt(data[0]), green = UInt(data[1]), blue = UInt(data[2]), alpha = UInt(data[3])
+        // test bottom because of potential rounded corners at the top
+        let bottom = Int(image.size.width) * Int(image.size.height) * 4
+        let red = UInt(data[bottom + 0]), green = UInt(data[bottom + 1]), blue = UInt(data[bottom + 2]), alpha = UInt(data[bottom + 3])
```

Comment:
iOS 13 modal navigation bars have rounded corners so we were testing against a black background. This gets at the bottom of the image which should contain the expected color.